obspy/imaging/waveform.py

```diff
@@ -214,6 +215,8 @@ def __init__(self, **kwargs):
         self.one_tick_per_line = kwargs.get('one_tick_per_line', False)
         self.show_y_UTC_label = kwargs.get('show_y_UTC_label', True)
         self.title = kwargs.get('title', self.stream[0].id)
+        #If true the signal wiglets will be filled
+        self.wiglets = kwargs.get('wiglets', True)
```

Comment:
Please make sure this defaults to False - this should also fix most test failures.